.github/workflows/static-analysis-php.yml

```
@@ -63,4 +68,9 @@ jobs:
           composer-options: ${{ inputs.COMPOSER_ARGS }}
 
       - name: Run Psalm
+        if: ${{ hashFiles('psalm.xml.dist', 'psalm.xml') != '' }}
         run: ./vendor/bin/psalm --php-version="${{ inputs.PHP_VERSION }}" ${{ inputs.PSALM_ARGS }}
+
+      - name: Run PHPStan
+        if: ${{ hashFiles('phpstan.neon.dist', 'phpstan.neon', 'phpstan.dist.neon') != '' }}
```

Please forgive me, but my grammar OCD forces me to sort the config files alphabetically. 🙈
```suggestion
        if: ${{ hashFiles('psalm.xml', 'psalm.xml.dist') != '' }}
        run: ./vendor/bin/psalm --php-version="${{ inputs.PHP_VERSION }}" ${{ inputs.PSALM_ARGS }}

      - name: Run PHPStan
        if: ${{ hashFiles('phpstan.dist.neon', 'phpstan.neon', 'phpstan.neon.dist') != '' }}
```